### DISCUSSION TITLE
after login i go back to the landing page

### DISCUSSION THREAD

**Discussion by mawnir** [gokulkrishh/bmrk.cc#10] [Q&A]
in localhost after login i go back to the landing page, the only thing i changed is :
```
await supabase.auth.signInWithOAuth({
      provider: 'github', <=== from google
      options: {
        redirectTo: urls.authCallback,
      },
    });
```
when i go to "http://app.localhost:3000/" i get "http://app.localhost:3000/account"
what am i doing wrong?

**Comment by mawnir** [comment]
I believe its a caching problem

**Comment by gokulkrishh** [comment]
There is redirection happening in layout page to check if user is logged in. See if the login refresh tokens are set properly in your local.

**Comment by mawnir** [comment]
`refresh tokens are set properly in your local.` where? in the browser? 
after login i get : http://localhost:3000/?code=10005d0a-2b7b-4359-b1c2-xxxxxxx
